src/index.js

```diff
@@ -7,7 +7,7 @@ const defaults = {
 };
 
 function syncEvent(node, eventName, newEventHandler) {
-  const eventNameLc = eventName.toLowerCase();
+  const eventNameLc = eventName.charAt(0).toLowerCase() + eventName.substring(1);
```

Comment:
Yeah, I'll change it.
originally copied from somewhere else. 
Reasons to use `charAt()` for reference only :)
`charAt()` is a getter only so it guarantees not to modify the string. Also browser support. 

*None of these is relevant here.*